### DISCUSSION TITLE
Add Dynamic Columns inside Filament Tables based On Resource Relations

### DISCUSSION THREAD

**Discussion by Ahsan-zaman** [filamentphp/filament#9134] [Help]
Hello Everyone 
I've recently started using the filamentPHP for some of my projects. (loving it so far, Great Tool👏)

I checked the documentation & searched online also, but couldn't find any useful resource regarding this.
I'm trying to build a table where the columns should by dynamic.
For Example "Parent A" has 3 children, So I want to add 3 columns for "Parent A".
Similarly "Parent B" has 1 child, SO only 1 column should be added.

I hope the context is clear 🙏🙏

Now my Question is 
#1 - Is this Even possible with V3 Table component ?
#2 - If not, Then what should be the best solution Livewire or basic Laravel ?

Thanks in advance 🤝

**Comment by sinnbeck** [comment]
I am having a hard time visualizing what you are after. Each row should have their own columns instead of one set of columns for the whole table or?

**Comment by leandrocfe** [comment]
You can't show columns by row, but you could show 4 columns with their data.
